src/index.js

```diff
@@ -14,7 +19,11 @@ import injectJSXWrapper from "./helpers/runtime/jsx-wrapper";
 
 export default function ({ Plugin, types: t }) {
   return new Plugin("incremental-dom", { visitor : {
-    Program: setupInjector,
+    Program: function(program, parent, scope, file) {
+      const options = get(file, ['opts', 'extra', 'incremental-dom']);
+      setPrefix(options);
+      setupInjector(program, parent, scope, file);
```

Comment:
Here's without the state: https://github.com/babel-plugins/babel-plugin-incremental-dom/commit/799b9ed60fd530be6558145a595ed24a3c84ca85